### ISSUE TITLE
Provide unversioned (vanity) releases page.

### ISSUE THREAD

**Issue by RokeJulianLockhart** [uutils/coreutils#5542]
It shall allow programs that use GitHub as their ultimate origin point for binaries to not worry about versions, instead always pulling the latest version. Not everything supports RegEx. For example, https://github.com/uutils/coreutils/issues/4291#issuecomment-1812814643.

**Comment by sylvestre** [comment]
Do you have example of project doing that ? thanks

**Comment by RokeJulianLockhart** [comment]
> https://github.com/uutils/coreutils/issues/5542#issuecomment-1812839663

@sylvestre,

0.	https://github.com/ClementTsang/bottom/releases/tag/nightly
0.	[x] https://github.com/Fleex255/PolicyPlus/issues/71#event-9828806901
	Although per https://github.com/Fleex255/PolicyPlus/issues/71#issuecomment-1636793346, the owner, @Fleex255, went about it in a different manner. You might prefer this, @YDX-2147483647. You can see this at https://github.com/Fleex255/PolicyPlus/releases.

There's also some useful discussion and positive reception to the issue at:

0.	[ ] https://github.com/PowerShell/PowerShell/issues/20338#issuecomment-1734273622
0.	[ ] https://github.com/illogical-robot/apkmirror-public/issues/231#issue-1329278729

**Comment by tertsdiepraam** [comment]
Right, lots of things to consider here I think. Let's hope I get what this is about: if I understand correctly, you want an easy way to download artifacts for the `main` branch? Or do you want the `release/latest` thing mentioned above?

First, let's consider the current state:
- We provide artifacts for each release (every 6 weeks or shorter)
- Artifacts for each CI build are available too, though hard to find and without a stable url. For example: https://github.com/uutils/coreutils/actions/runs/6877396975
- Like mentioned above, there is a url for accessing the latest release.

We could add a page to the website with the latest artifacts or something like that? How would you propose we implement this?

**Comment by RokeJulianLockhart** [comment]
@tertsdiepraam,

> We could add a page to the website with the latest artifacts or something like that? How would you propose we implement this?

Probably how https://github.com/ClementTsang/bottom/releases/ does — https://github.com/ClementTsang/bottom/releases/tag/nightly.

**Comment by tertsdiepraam** [comment]
Is that generated every day? What does that provide to you over the normal releases?

**Comment by RokeJulianLockhart** [comment]
> https://github.com/uutils/coreutils/issues/5542#issuecomment-1813442092

No, @tertsdiepraam, I don't want to it to be generated each day, though I don't particularly care if it works (nightly builds obviously probably wouldn't though). I'm providing an example of an unversioned release which is updated at regular intervals, as you requested.

**Comment by tertsdiepraam** [comment]
Alright, I understand now! Thanks! So, the `release/latest` link is not enough?

**Comment by RokeJulianLockhart** [comment]
I... wasn't aware that existed, @tertsdiepraam. In a bout of intense shame, I pass the baton of whether to close this issue onto you the participants here. There might yet be rationale for having a dedicated latest release like https://github.com/ClementTsang/bottom/releases/tag/nightly does, but I can't imagine it being more useful than https://github.com/uutils/coreutils/releases/latest.

**Comment by RokeJulianLockhart** [comment]
Aha! I just found the rationale whilst testing furiously, attempting to regain my lost honour:

Although is useful for a human, linking to specific assets is unsupported even when the URI correlates to its version. For instance, https://github.com/uutils/coreutils/releases/latest/coreutils-0.0.23-x86_64-pc-windows-msvc.zip returns 404. Additionally, because the assets are currently versioned too in their filenames, vanity filenames would be necvessary in the unversioned release too.

**Comment by tertsdiepraam** [comment]
Oh yeah that makes sense :)

**Comment by YDX-2147483647** [comment]
> For instance, [uutils/coreutils/releases/latest/coreutils-0.0.23-x86_64-pc-windows-msvc.zip](https://github.com/uutils/coreutils/releases/latest/coreutils-0.0.23-x86_64-pc-windows-msvc.zip) returns 404.

Linking to specific assets **is supported**. It's _404 Not Found_ because `/download` is missing — https://github.com/uutils/coreutils/releases/latest/download/coreutils-0.0.23-x86_64-pc-windows-msvc.zip returns _302 Found_ and redirects to  https://github.com/uutils/coreutils/releases/download/0.0.23/coreutils-0.0.23-x86_64-pc-windows-msvc.zip . (Your conclusion still holds, however)

**Comment by oech3** [comment]
My PR is not working :|
